mathics/builtin/files.py

```diff
@@ -2531,21 +2518,19 @@ class ToFileName(Builtin):
     """
     <dl>
     <dt>'ToFileName[{"$dir_1$", "$dir_2$", ...}]'
-      <dd>joins the $dir_i$ togeather into one path.
+      <dd>joins the $dir_i$ together into one path.
     </dl>
 
     'ToFileName' has been superseded by 'FileNameJoin'.
 
-    #> Unprotect[$PathnameSeparator]; $PathnameSeparator = "/"; Protect[$PathnameSeparator];
-
     >> ToFileName[{"dir1", "dir2"}, "file"]
-     = dir1/dir2/file
+     = dir1...dir2...file
 
     >> ToFileName["dir1", "file"]
-     = dir1/file
+     = dir1...file
 
     >> ToFileName[{"dir1", "dir2", "dir3"}]
-     = dir1/dir2/dir3
+     = dir1...dir2...dir3
```

Comment:
`...` would be also a good way to deal with digits beyond accuracy in numeric tests!